core/base/persistenceDiagramClustering/PersistenceDiagramClustering.h

```diff
@@ -0,0 +1,411 @@
+#ifndef _PERSISTENCEDIAGRAMSCLUSTERING_H
```

Comment:
Thanks for adding authorship and references.

Reply:
All done! Also there shouldn't be any warnings anymore.
cheers,
Jules